packages/components/src/components/Image/Image.tsx

```diff
@@ -18,7 +18,9 @@ export const allowedImageFrameProps = [
     'height',
     'flex',
 ] as const satisfies FramePropsKeys[];
-type AllowedFrameProps = Pick<FrameProps, (typeof allowedImageFrameProps)[number]>;
+type AllowedFrameProps = Pick<FrameProps, (typeof allowedImageFrameProps)[number]> & {
```

Comment:
We do not extend FrameProps like this

cc @jvaclavik

Reply:
Yes, Tomáš is right, please move it to ImageProps. Like:

Line 67:
```tsx
export type ImageProps = AllowedFrameProps &
    ImageHTMLProps &
    (
        | {
              image: ImageKey;
              imageSrc?: never;
          }
        | {
              image?: never;
              imageSrc: string;
          }
    ) & { isFilterActive?: boolean };

Reply:
See the rotate animation up in the pill and image itself.

<img width="525" height="651" alt="Screenshot 2025-10-01 at 11 57 15 AM" src="https://github.com/user-attachments/assets/b6de1692-54fa-46e7-bcf6-c9e2a00dec98" />


Down here it is without the filter activated 
<img width="526" height="705" alt="Screenshot 2025-10-01 at 12 14 55 PM" src="https://github.com/user-attachments/assets/05830597-dca0-4f56-abee-95fa17deef73" />

Reply:
@jvaclavik can we validate that this global introduction of the filter to all StyledImages is correct or not? 

Like we encoutered it with @MiroslavProchazka and wer are quite confused.

Reply:
I have created a PR that disables this global filter at least for teh device images https://github.com/trezor/trezor-suite/compare/confirm-modal-renders?expand=1

But I am not sure where esle thise should be disabled.

IT seems to me, that this filter should have been turned on only for the images in this PR, or?

Reply:
Yes, we discussed this issue, you are right, we have to change these renders.